src/Node/Expression/CallExpression.php

```diff
@@ -306,12 +314,18 @@ private function reflectCallable($callable)
             $class = substr($callable, 0, $pos);
             $method = substr($callable, $pos + 2);
             if (!method_exists($class, $method)) {
+                if (!method_exists($class, '__callStatic')) {
```

Comment:
Should this also check for __call? We allow [FQCN, 'nonStaticMethod'] for instance methods even though that is not a legal callable. Is 'FQCN::nonStaticMethod' also legal in context of Twig?

Reply:
@stof What do you think?